india_compliance/gst_india/doctype/bill_of_entry/bill_of_entry.py

```diff
@@ -102,6 +104,9 @@ def set_total_taxes(self):
             if tax.charge_type == "On Net Total":
                 tax.tax_amount = self.get_tax_amount(tax.item_wise_tax_rates)
 
+            if tax.account_head in frappe.flags.round_off_applicable_accounts:
+                tax.tax_amount = round(tax.tax_amount, 0)
```

Comment:
```suggestion
	            if tax.account_head in frappe.flags.round_off_applicable_accounts:
	                tax.tax_amount = round(tax.tax_amount, 0)
```

Should be ideally done only for auto-calculated values and not actual values. The same correction can be done in JS.
As we only have On Net Total and Actual, only applies to On Net Total